index.html

```diff
@@ -5279,17 +5279,35 @@ <h2 id="defining-a-valid-audioworkletprcoessor">
             </p>
             <ol>
               <li>
-                <code>input</code> of type
-                <code>sequence&lt;sequence&lt;Float32Array&gt;&gt;</code><br>
-                The input audio buffer from the incoming connections provided
-                by the user agent. <code>inputs[n][m]</code> is a
-                <code>Float32Array</code> of audio samples for the
-                <code>m</code>th channel of <code>n</code>th input. While the
-                number of inputs is fixed at the construction, the number of
-                channels can be changed dynamically.
+                <p>
+                  <code>inputs</code> of type
```

Comment:
I don't think it is possible to use the usual dd/dt markup, because this is not IDL (as per TAG feedback).